src/components/CCPDashboard/PatientInfoTable.tsx

```diff
@@ -212,137 +230,291 @@ export const PatientInfoTable = ({
     setOpenDeletePatient(false);
   };
 
-  const tableRows = stableSort(patients, getComparator(order, orderBy)).map(
-    (patient: Patient) => {
-      const triageLevels = {
-        [TriageLevel.GREEN]: {
-          colour: Colours.TriageGreen,
-          triageLevel: TriageLevel.GREEN,
-          label: 'Green',
-        },
-        [TriageLevel.YELLOW]: {
-          colour: Colours.TriageYellow,
-          triageLevel: TriageLevel.YELLOW,
-          label: 'Yellow',
-        },
-        [TriageLevel.RED]: {
-          colour: Colours.TriageRed,
-          triageLevel: TriageLevel.RED,
-          label: 'Red',
-        },
-        [TriageLevel.BLACK]: {
-          colour: Colours.Black,
-          triageLevel: TriageLevel.BLACK,
-          label: 'Black',
-        },
-        [TriageLevel.WHITE]: {
-          colour: Colours.BorderLightGray,
-          triageLevel: TriageLevel.WHITE,
-          label: 'White',
-        },
-      };
+  const handleConfirmRestorePatient = () => {
+    history.push(
+      `/events/${eventId}/ccps/${ccpId}/patients/${
+        ((selectedPatient as unknown) as Patient).id
+      }`
+    );
+    setOpenRestorePatient(false);
+  };
+
+  const handleCancelRestorePatient = () => {
+    setOpenRestorePatient(false);
+  };
+
+  const handleClickRestore = () => {
+    setAnchorElRestore(null);
+    setOpenRestorePatient(true);
+  };
+
+  const tableRows = stableSort(
+    activePatients,
+    getComparator(order, orderBy)
+  ).map((patient: Patient) => {
+    const triageLevels = {
+      [TriageLevel.GREEN]: {
+        colour: Colours.TriageGreen,
+        triageLevel: TriageLevel.GREEN,
+        label: 'Green',
+      },
+      [TriageLevel.YELLOW]: {
+        colour: Colours.TriageYellow,
+        triageLevel: TriageLevel.YELLOW,
+        label: 'Yellow',
+      },
+      [TriageLevel.RED]: {
+        colour: Colours.TriageRed,
+        triageLevel: TriageLevel.RED,
+        label: 'Red',
+      },
+      [TriageLevel.BLACK]: {
+        colour: Colours.Black,
+        triageLevel: TriageLevel.BLACK,
+        label: 'Black',
+      },
+      [TriageLevel.WHITE]: {
+        colour: Colours.BorderLightGray,
+        triageLevel: TriageLevel.WHITE,
+        label: 'White',
+      },
+    };
 
-      const statusLabels = {
-        [Status.ON_SITE]: 'On Scene',
-        [Status.TRANSPORTED]: 'Transported',
-        [Status.RELEASED]: 'Released',
-        [Status.DELETED]: 'Omitted/Deleted',
-      };
+    const statusLabels = {
+      [Status.ON_SITE]: 'On Scene',
+      [Status.TRANSPORTED]: 'Transported',
+      [Status.RELEASED]: 'Released',
+      [Status.DELETED]: 'Omitted/Deleted',
```

Comment:
According to the Figma, it seems this has been updated to say just 'Deleted'

Reply:
Will change, thank you for catching that.